server/src/services/tooljet_db.service.ts

```diff
@@ -610,12 +610,20 @@ export class TooljetDbService {
 
     if (!internalTable) throw new NotFoundException('Internal table not found: ' + tableName);
 
-    const query = `ALTER TABLE "${internalTable.id}" DROP COLUMN ${column['column_name']}`;
+    // const query = `ALTER TABLE "${internalTable.id}" DROP COLUMN "${column['column_name']}"`;
+    const tjdbQueryRunnner = this.tooljetDbManager.connection.createQueryRunner();
+    await tjdbQueryRunnner.connect();
+    await tjdbQueryRunnner.startTransaction();
+
     try {
-      const result = await this.tooljetDbManager.query(query);
+      const result = await tjdbQueryRunnner.dropColumn(internalTable.id, column['column_name']);
+      await tjdbQueryRunnner.commitTransaction();
       await this.tooljetDbManager.query("NOTIFY pgrst, 'reload schema'");
+      await tjdbQueryRunnner.release();
       return result;
     } catch (error) {
+      await tjdbQueryRunnner.rollbackTransaction();
+      await tjdbQueryRunnner.release();
```

Comment:
Since there is only one query, we don't have to start a transaction. Transaction is only needed when we have to rollback multiple queries.

Actually, you can do something like:

```js
try {
  const result = await this.tooljetDbManager.queryRunner.dropColumn(internalTable.id, column['column_name']);

  await this.tooljetDbManager.query("NOTIFY pgrst, 'reload schema'");
  return result;
} catch(error) {
......

```